softwareComponents/configuration/include/configuration/joints.hpp

```diff
@@ -49,11 +51,58 @@ struct Joint: public atoms::VisitableBase< Joint, JointVisitor > {
         return _positions;
     }
 
+    /**
+     * @brief Sets joint settings to corresponding values given by <pos>.
+     * Assumes there are exactly as many values as there are joint settings.
+     * 
+     * @throws std::logic_error if the given settings do not respect joint limits.
+     */
     void setPositions( std::span< const float > pos ) {
-        assert( pos.size() == _positions.size() && "Positions have to preserve given size" );
+        assert( pos.size() == _positions.size() && "Insufficient number of parameters to set" );
```

Comment:
On to není jen "insufficient", ale také excessive.

Reply:
Přepsáno na "incorrect".